github/repos_hooks.go

```diff
@@ -224,3 +249,27 @@ func (s *RepositoriesService) TestHook(ctx context.Context, owner, repo string,
 	}
 	return s.client.Do(ctx, req, nil)
 }
+
+// CreatePubSubHubBubrequest lets servers register receive updates when a topic is updated
+// using publish/subscribe protocol with GitHub as the hub for the repositories.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/hooks/#pubsubhubbub
+func (s *RepositoriesService) CreatePubSubHubBubRequest(ctx context.Context, request *PubSubHubBubRequest) (*Response, error) {
```

Comment:
Let's change `request *PubSubHubBubRequest` to:
`opts *SubscribePubSubHubbubOpts`.